src/config.rs

```diff
@@ -254,6 +255,36 @@ impl User {
     }
 }
 
+/// TLS connection mode to the server.
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
+#[serde(untagged)]
+pub enum CertificateVerificationVariant {
+    Bool(bool),     // false -> prefer, true - verify-full
+    String(String), // "only-ca" -> verify-full
```

Comment:
"verify-ca" is an actual mode supported by the client, we should call it that way here too: https://www.postgresql.org/docs/current/libpq-ssl.html

Reply:
> "verify-ca" is an actual mode supported by the client, we should call it that way here too: https://www.postgresql.org/docs/current/libpq-ssl.html

I'll correct that oversight